src/content/tools/pubspec.md

```diff
@@ -21,46 +21,45 @@ needs to know. The pubspec is written in
 [YAML][], which is human readable, but be aware
 that _white space (tabs v spaces) matters_.
 
-[YAML]: https://yaml.org/
+The pubspec specifies dependencies
+that the project requires, such as:
 
-The pubspec file specifies dependencies
-that the project requires, such as particular packages
-(and their versions), fonts, or image files.
-It also specifies other requirements, such as 
-dependencies on developer packages (like
-testing or mocking packages), or particular
-constraints on the version of the Flutter SDK. 
++ Particular packages and their versions
++ Fonts
++ Images
```

Comment:
Observation: developers can bundle any type of file, not just fonts and images. Having said that, I think it is appropriate to keep things simple in an introduction like this 👍

Reply:
I also see we cover this earlier, so disregard :)